logger/logger.rb

```diff
@@ -1,3 +1,10 @@
+require './lib/slack'
+
 require './lib/slack_logger'
+require './lib/slack_events_receiver'
+require './lib/slack_rtm_receiver'
+
+config = YAML.load_file('./config.yml')
 
-SlackLogger.new.start
+receiver = Slack::Events.config.signing_secret ? SlackEventsReceiver.new(config['slack']['team_id']) : SlackRTMReceiver.new
```

Comment:
[nit]

```suggestion
receiver = config['slack']['event_api'] ? SlackEventsReceiver.new(config['slack']['team_id']) : SlackRTMReceiver.new
```

のほうがすき (まあべつにどっちでも)

Reply:
ぶどう！